Cargo.toml

```diff
@@ -33,7 +33,9 @@ default = ["pem"]
 features = ["x509-parser"]
 
 [dev-dependencies]
+native-tls = "0.2"
```

Comment:
If we do this with rustls we can don't even need the pipe crate, and we can avoid relying on C libraries.

(Honestly, I'm not sure it makes sense to include a whole TLS setup in the example -- that seems to be more like a testing thing?)

Reply:
In my case the 'solution to a problem' was 'how can i create temporary servers for testing with temporary TLS/SSL certificates, and connect clients without suppressing TLS/SSL validations' - a common testing scenario.